geometry_msgs/Point position  # Position where the attack is initiated
std_msgs/Header header        # Includes timestamp to manage the attack duration

